RestInfrastructure/Authenticators/ApiReadAuthenticator.cs

```diff
@@ -0,0 +1,51 @@
+﻿using RestSharp;
+using RestSharp.Authenticators;
+using Task10.RestInfrastructure.DataModels;
+
+namespace Task10.RestInfrastructure.Authenticators
+{
+    internal class ApiReadAuthenticator : AuthenticatorBase
+    {
+        readonly string _baseUrl;
+        readonly string _clientId;
+        readonly string _clientSecret;
+
+        public ApiReadAuthenticator() : base("")
+        {
+            _baseUrl = "http://localhost:49000";
```

Comment:
I would store this data in some separate class or file to avoid duplicating

Reply:
I have added Settings folder to manage different settings. I am storing this data in appsettings.json file now. Please check this out.